app/code/Magento/Sitemap/Api/Data/SitemapInterface.php

```diff
@@ -0,0 +1,121 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\Sitemap\Api\Data;
+
+interface SitemapInterface extends SitemapExtensionInterface
```

Comment:
Please, extend current interface from \Magento\Framework\Api\ExtensionAttributesInterface

Reply:
Made it extend the right thing